e2e/flags.go

```diff
@@ -257,12 +257,32 @@ const (
 	DefaultGKEChannel = "regular"
 	// DefaultGKEMachineType is the default GKE machine type to use when creating a cluster
 	DefaultGKEMachineType = "n2-standard-8"
+	// DefaultGKEMachineTypeForStressTests is the default GKE machine type to
```

Comment:
fyi if you want these to be used by the prow jobs you'll need to update the config: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml#L383

Reply:
thanks. we should probably just remove the GKE_MACHINE_TYPE there, since we had the same as default anyway

Reply:
I think we'll want to merge this first, then we can remove the env vars from the prow jobs

Reply:
Looks like the stress tests inherit from the non-stress config, which makes it hard to remove the defaults for stress tests without removing the standard default. So I might have to make another PR for that.

Reply:
Plus, it looks like prow already has standard clusters using `n2-standard-4` nodes with 25Gb disk. So The defaults in the code are out of date.

Reply:
https://github.com/GoogleCloudPlatform/oss-test-infra/pull/2449